src/runlib.rs

```diff
@@ -353,6 +403,13 @@ mod test {
                 b"7d865e959b2466918c9863afca942d0fb89d7c9ac0c99bafc3749504ded97730",
             ),
         );
+        expected.insert(
+            VirtualTargetPath::new("tests/test_runlib/.hidden/world".to_string()).unwrap(),
+            create_target_description(
+                crypto::HashAlgorithm::Sha256,
+                b"25623b53e0984428da972f4c635706d32d01ec92dcd2ab39066082e0b9488c9d",
+            ),
+        );
```

Comment:
I may be missing something here. Do we need to insert this path into this, admittedly old test?

Reply:
It is used for test left-strip for same name files `hello./world` and `.hidden/world`.
Obviously, we need two files with same name and different prefix path.

If it is not a good idea to add a file in exist test path, whether it's ok to add a new test directory 'tests/test_prefix/*' annd add some files in it?